### DISCUSSION TITLE
Is there a way to sync/replicate Multiple Security Onion deployments (elastic search indexes)

### DISCUSSION THREAD

**Discussion by Leche671** [Security-Onion-Solutions/securityonion#6861] [Unsupported Versions] [answered]
I am trying to find if there is a feasible solution to forward or replicate Security Onion indexes from an existing deployment to another Security Onion deployment.

Is there a "beat" app that can forward to an external SecOnion server and have it ingested properly for instance.

**Comment by abesinger** [comment, marked as answer]
I suspect you want to look at [Cross Cluster Replication](https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-ccr.html)
